### PR TITLE
UPSE-288: more Java 11 updates

### PR DESCRIPTION
**Pull request by groybal**
More Java 11 updates:
* fix logging
* add Java 11 to github build
* fix dom4j NoSuchMethodError issue due to two versions of dom4j jars in WEB-INF/lib

### CODE REVIEW

**Comment by groybal** [comment]
How do we handle the sonatype-lift issues?

File: pom.xml [hunk 1]
```diff
@@ -175,11 +183,23 @@
                 <groupId>commons-httpclient</groupId>
                 <artifactId>commons-httpclient</artifactId>
                 <version>3.1</version>
```
[the remark this answers is not in the document]
Comment (ChristianMurphy) [reply pom.xml:185]: More broadly we probably want to migrate to the new maven package for httpclient
v3: https://mvnrepository.com/artifact/commons-httpclient/commons-httpclient (currently used)
v4: https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient
v5: https://mvnrepository.com/artifact/org.apache.httpcomponents.client5/httpclient5

But that may be outside the scope of the Java 11 upgrade

File: pom.xml [hunk 2]
```diff
@@ -327,26 +359,26 @@
                 <version>1.1.2</version>
```
[the remark this answers is not in the document]
Comment (ChristianMurphy) [reply pom.xml:359]: This package has moved in maven central.
We probably want switch to the new package https://mvnrepository.com/artifact/org.apache.taglibs/taglibs-standard-impl at some point.
That may not be needed as part of the Java 11 upgrade.

**Comment by ChristianMurphy** [comment]
> How do we handle the sonatype-lift issues?

If you have some time available, it would be great to upgrade those dependencies.
But that doesn't strictly need to happen as part of the Java 11 upgrade, note them down for later review/resolution.

**Review by loulou2u — APPROVED** [review]
Approved for Java 11 scope, but created issue #229 to address migrating to new maven package for httpclient. Thanks Christian and Gary!

File: pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (loulou2u) [reply pom.xml:359]: Created issue #230 to handle this one too! Thanks Christian

**Pull request merged**